cmd/store/store.go

```diff
@@ -748,37 +748,44 @@ func (s *Store) GetContext() (*Context, error) {
 	return &c, nil
 }
 
-func (ctx *Context) ResolveKey(kind nkeys.PrefixByte, flagValue string) (nkeys.KeyPair, error) {
+func (ctx *Context) ResolveKey(flagValue string, kinds ...nkeys.PrefixByte) (nkeys.KeyPair, error) {
```

Comment:
It would only be wrong if you put them in the wrong order. if you say operator then account all is well.
In  cmd/signerparams.go  line 39 (only time we use more than two entries) this is what is set up.
I tried invoking this twice, did not go well...

Reply:
I would sort the keys so that operator is always guaranteed to be first regardless of what the caller does. This I think would be right behavior. Account is signed by operator if we have the key, otherwise it will keep looking.

Reply:
line 779 takes care of that. no matter how the keys are sorted, this will assure that we return them in the order determined by kind